### PR TITLE
tikv: add a new config `server.simplify-metrics`

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -200,6 +200,11 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 该配置项指定 TiKV 中发送 Raft 消息的缓冲区大小。如果存在消息发送不及时导致缓冲区满、消息被丢弃的情况，可以适当调大该配置项值以提升系统运行的稳定性。
 + 默认值：8192
 
+### `simplify-metrics`
```
Comment (TomShawn) [inline tikv-configuration-file.md:203]:
```suggestion
### `simplify-metrics` <span class="version-mark">从 v6.2.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -200,6 +200,11 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
+
++ 是否开启精简返回的 Metrics 数据。在设置为 true 时，TiKV 可以通过过滤部分无用的 Metrics 采样数据以减少每次请求返回的 Metrics 数据量。
```
Comment (TomShawn) [inline tikv-configuration-file.md:205]: 是否支持在线修改呢？

Comment (TomShawn) [reply tikv-configuration-file.md:205]: Metrics 数据是集群监控数据吗？
```suggestion
+ 是否精简返回的 Metrics 数据。设置为 `true` 后，TiKV 可以通过过滤部分 Metrics 采样数据以减少每次请求返回的 Metrics 数据量。
```

Comment (glorv) [reply tikv-configuration-file.md:205]: 是的

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -200,6 +200,11 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 该配置项指定 TiKV 中发送 Raft 消息的缓冲区大小。如果存在消息发送不及时导致缓冲区满、消息被丢弃的情况，可以适当调大该配置项值以提升系统运行的稳定性。
 + 默认值：8192
 
+### `simplify-metrics` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
++ 是否精简返回的 Metrics 数据。设置为 `true` 后，TiKV 可以通过过滤部分 Metrics 采样数据以减少每次请求返回的 Metrics 数据量。
```
Comment (TomShawn) [inline tikv-configuration-file.md:205]:
```suggestion
+ 是否精简返回的监控指标 Metrics 数据。设置为 `true` 后，TiKV 可以通过过滤部分 Metrics 采样数据以减少每次请求返回的 Metrics 数据量。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by glorv** [comment]
@BusyJay PTAL

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -200,6 +200,11 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 该配置项指定 TiKV 中发送 Raft 消息的缓冲区大小。如果存在消息发送不及时导致缓冲区满、消息被丢弃的情况，可以适当调大该配置项值以提升系统运行的稳定性。
 + 默认值：8192
 
+### `simplify-metrics` <span class="version-mark">从 v6.2.0 版本开始引入</span>
```
Comment (BusyJay) [inline tikv-configuration-file.md:203]: Perhaps `filter-metrics` is a better name. Though I don't have strong opinions.

**Review by BusyJay — APPROVED** [review]

**Pull request merged**